app/views/declaration/additionaldeclarationtype/declaration_type.scala.html

```diff
@@ -27,9 +27,10 @@
     title = messages("declaration.declarationType.title")
 ) {
 
-    @components.back_link(controllers.declaration.routes.DispatchLocationController.displayPage(mode), mode)
+    @components.back_link(controllers.declaration.routes.DispatchLocationController.displayPage(), request.mode)
```

Comment:
if back_link would take `(mode: Mode) => call` this would even simplify to
``` 
@components.back_link(controllers.declaration.routes.DispatchLocationController.displayPage)
```
Even having mode in controller signature
Even if controller would have more then one parameter 
```
@components.back_link(MuplitpleParamterPath.displayPage(_, otherParamter)) 
```
still look good